ts/features/bonus/cdc/analytics/index.ts

```diff
@@ -15,3 +19,11 @@ export const trackCdcGoToService = () =>
 
 export const trackCdcCardError = () =>
   mixpanelTrack("CDC_CARD_ERROR", buildEventProperties("KO", "error"));
+
+export async function* trackCdcStatus() {
+  const state: GlobalState = yield* select();
+  await updateMixpanelProfileProperties(state, {
+    property: "CDC_STATUS",
+    value: cdcStatusHandler(state)
+  });
+}
```

Comment:
Keep it as a standard generator (function*) for Saga compatibility, and use void to trigger async side effects without blocking the generator

```suggestion
export const trackCdcStatus = function* () {
  const state: GlobalState = yield* select();
  void updateMixpanelProfileProperties(state, {
    property: "CDC_STATUS",
    value: cdcStatusHandler(state)
  });
};
```